### ISSUE TITLE
Cherry pick commits from Saul's omni

### ISSUE THREAD

**Issue by adampbeardsley** [HERA-Team/heracal#16]
make on heracal branch, cherry pick from capo.omni, issue pull request

**Comment by SaulAryehKohn** [comment]
There is now a `polDev` branch on this repo, in which I've just pushed my changes to `heracal/omni.py` and `heracal/tests/test_omni.py`. The build passes on Travis CI and test coverage goes up by 2%. I'm waiting on Issue #11 to be resolved before I start editing scripts `omni_run` and `omni_apply` for polarization capabilities. Should I issue pull request ASAP or wait for #11 ?

**Comment by bhazelton** [comment]
I think it depends on whether the edits you've already made are useful on their own or whether they really need the changes in `omni_run` and `omni_apply` to be useful. I don't have a strong opinion.

**Comment by SaulAryehKohn** [comment]
This is accomplished by Pull Request #19

**Comment by SaulAryehKohn** [comment]
These have been merged into the master branch.
